functional-test/src/test/java/org/zanata/feature/project/EditProjectLanguagesTest.java

```diff
@@ -129,4 +130,61 @@ public void setLanguageAliasTest() {
                 .isEqualTo("pl-PL")
                 .as("The alias was set");
     }
+
+    @Feature(summary = "The administrator can remove an alias for a project " +
+            "language",
+            tcmsTestPlanIds = 5316, tcmsTestCaseIds = 0)
+    @Test
+    @Ignore("RHBZ1199371")
+    public void removeLanguageAliasTest() {
+        ProjectLanguagesTab projectLanguagesTab = new ProjectWorkFlow()
+                .goToProjectByName("about fedora")
+                .gotoSettingsTab()
+                .gotoSettingsLanguagesTab()
+                .clickLanguageActionsDropdown("pl")
+                .clickAddAlias("pl")
+                .enterAliasForLocale("pl", "pl-PL")
+                .saveLocaleAlias("pl");
+
+        assertThat(projectLanguagesTab.getAlias("pl"))
+                .isEqualTo("pl-PL")
+                .as("The alias was set");
+
+        projectLanguagesTab = projectLanguagesTab
+                .clickLanguageActionsDropdown("pl")
+                .removeLocale("pl");
+
+        assertThat(projectLanguagesTab.getAlias("pl"))
+                .isEmpty();
```

Comment:
I think this just looks in active locales, so it is only showing that locale "pl" has been removed. It looks like it is trying to check that there is no alias associated with "pl". To check that you could remove then re-add it, then check.